bin/ansible-test

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+# PYTHON_ARGCOMPLETE_OK
+"""Primary entry point for ansible-test."""
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+import os
+import sys
+
+if __name__ == '__main__':
+    sys.path.insert(0, os.path.abspath(os.path.join(__file__, '..', '..', 'test', 'runner')))
```

Comment:
Shouldn't this go one down, instead of two down ?

Reply:
Hmm, I guess it uses the filename as well. That's why I hate that notation and I avoided it :)
Doing os.path.dirname() twice is easier to read IMO.

Reply:
I'm happy to change it if it makes it easier to read.

Reply:
I am a fan of `os.path.dirname` as well.